crates/autopilot/src/database/quotes.rs

```diff
@@ -16,6 +19,20 @@ impl Postgres {
         database::quotes::remove_expired_quotes(&mut ex, max_expiry).await?;
         Ok(())
     }
+
+    pub async fn read_quotes(&self, auction: &Auction) -> Result<HashMap<OrderUid, Quote>> {
+        let mut ex = self.pool.acquire().await?;
+        let mut quotes = HashMap::new();
+        for order in &auction.orders {
+            let order_uid = ByteArray(order.metadata.uid.0);
+            let quote = database::orders::read_quote(&mut ex, &order_uid)
+                .await?
+                .context("failed to read quote")?;
```

Comment:
This can happen (cf. this [oncall issue](https://cowservices.slack.com/archives/C0371SB243E/p1703527547249929?thread_ts=1703462512.028329&cid=C0371SB243E)) so we should probably not fail the entire method for it

Reply:
You are right! Fixed error handling. Now the whole fee policy building fails only if the connection to db failed.